process/README.md

```diff
@@ -0,0 +1,132 @@
+# I. Overview
+
+This policy describes the Open Source Security Foundation (OpenSSF) project life cycle process, from sandbox to archival and more. It describes the stages a project may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a project to move from one stage to another. It also describes the Annual Review Process through which those changes will be evaluated and made.
+
+Project progression, movement from one stage to another, allows projects to participate at the level that is most appropriate for them given where they are in their lifecycle.
+
+
+# II. Stages - Definitions & Expectations
+
+OpenSSF projects have a maturity level of sandbox, incubating, or graduated. Archived is for projects no longer in active development. The maturity level is a signal by OpenSSF as to what sorts of enterprises should be adopting different projects. Projects increase their maturity by demonstrating their sustainability to OpenSSF’s Technical Advisory Committee(TAC): that they have adoption, a healthy rate of changes, committers from multiple organizations, have adopted the OpenSSF Code of Conduct, and have achieved and maintained the Core Infrastructure Initiative Best Practices Badge.
+
+![Project Stages](https://github.com/ossf/tac/blob/main/process/project-stages.png)
+
+## Sandbox:
+
+The OpenSSF Sandbox is the entry point for early stage projects and has four goals:
+
+* Encourage public visibility of experiments or other early work that can add value to the OpenSSF mission and build the ingredients of a successful Incubation level project
+* Facilitate alignment with existing projects if (and only if) this is desired
+* Nurture projects (e.g. via OpenSSF Service Desk requests)
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct and IP Policy requirements
```

Comment:
Add sandbox requirement:

- Its README must say where it lives / reports to (e.g., if it's part of a WG, which WG; if it's a top-level project, that it's a top-level project & reports to the TAC).
- The README of the "owning" organization (WG or TAC) must add to its list.

That way it's obvious "who is where". That should be done very early. It can be changed over time, but we need to make it clear.